body/stretch_body/pimu.py

```diff
@@ -170,7 +170,9 @@ def __init__(self, event_reset=False):
         self._dirty_trigger = False
         self.frame_id_last = None
         self.frame_id_base = 0
-        self.transport = Transport(usb='/dev/hello-pimu', logger=self.logger)
+        if usb is None:
+            usb='/dev/hello-pimu'
```

Comment:
Instead of hard-coding the usb alias here, let's make it a parameter of "pimu" in the robot_params_*_.py dictionaries. This gives users the ability to override in user YAML.

Reply:
It is a good idea and has been implemented with all the devices.